proposals/unified-site.md

```diff
@@ -0,0 +1,163 @@
+# Unified W3DT docs & website
+
+Authors: johndmulhausen
+
+Initial PR: #5
+
+## Purpose &amp; impact
+#### Background &amp; intent
+
+Currently all information about our "stack" is siloed across multiple websites
+that are in various states of maintenance. There is no place to learn how the
+various pieces of the stack such as libp2p, IPFS, Filecoin, IPNS, etc work
+together. There is also no place where the benefits of the W3DT stack can be
+expressed in a definitive and reliable place. The tech that drives our site
+(built on Vuepress) has to be kept in sync manually across multiple websites.
+This way, one code push elevates the documentation and user experience across
+all our offerings and the popularity of one offering can "boost" another as we
+begin to show the integration points.
+
+Building a mental model of Web3 development is a primary adoption barrier for us,
+one that pairs very poorly with the difficulty inherent in experimenting with the tech
+itself. We need a place where we talk with a unified voice about how it's
+different, what its benefits are, why you would and wouldn't use it, how it
+compares to Web2, what good metaphors are in the current world for where we
+envision things going, and how the tech works together to address common
+scenarios in a unique way.
+
+Onboarding is an art, and creating a golden path through our offerings that
+shows various pieces of tech working together will require all of us pulling on
+the same oar, not continuing to maintain our silos.
+
+To continue to sink development time into any one particular website - such as,
```

Comment:
Each of our projects **must** continue to have a strong identity and entrypoint. Removing our project websites is antithetical to our goals - many folks can and will use and onboard on these projects in isolation and that is a completely acceptable onboarding point. Our work on making these protocols work well together is additive to the project identities themselves.